intervals/interval.py

```diff
@@ -321,7 +324,7 @@ def length(self):
     def radius(self):
         if self.length == inf:
             return inf
-        return float(self.length) / 2
+        return self.length / 2
```

Comment:
This change breaks tests on Python 2.7 and PyPy. Now radius returns int instead of float on those.